test/Base_Tests/src/Data/Vector_Spec.enso

```diff
@@ -1243,6 +1244,11 @@ add_specs suite_builder =
             [Nothing].pretty.should_equal "[Nothing]"
             [True, False, 'a'].pretty . should_equal "[True, False, 'a']"
             [Foo.Value True].pretty . should_equal "[(Foo.Value True)]"
+            [Date.new 2022 1 1].pretty . should_equal "[Date.new 2022 1 1]"
+
+            mixed = [1, 2, 'a', (Foo.Value True), Date.new 2022 1 1, Nothing]
+            mixed.pretty . should_equal "[1, 2, 'a', (Foo.Value True), Date.new 2022 1 1, Nothing]"
+            Debug.eval (mixed.pretty) . should_equal [1, 2, 'a', Foo.Value True, Date.new 2022 1 1, Nothing]
```

Comment:
Same for Array_Spec?

Reply:
Yep, I still need to add tests for Column and Table, too. I will do it before I merge.